packages/ra-ui-materialui/src/input/BooleanInput.tsx

```diff
@@ -111,11 +111,10 @@ BooleanInput.propTypes = {
     ...InputPropTypes,
     // @ts-ignore
     options: PropTypes.shape(Switch.propTypes),
-    disabled: PropTypes.bool,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏why remove this line?